README.md

```diff
@@ -38,14 +38,14 @@ START RequestId: 6a3bc71b-e369-498c-849c-f522e79ce734 Version: $LATEST
 ```
 - To kick off a port scan on a target on demand, do:
 ```
-$ curl --header "x-api-key: <API-KEY> -X PUT -d '{"target": "www.smh.com.au"}' https://<YOUR-API-ENDPOINT>/dev/ondemand/portscan`
+$ curl --header "x-api-key: <API-KEY> -X POST -d '{"target": "www.smh.com.au"}' https://<YOUR-API-ENDPOINT>/dev/ondemand/portscan`
```

Comment:
I  think ultimately, we should change  the readme to reflect the example cases we document to avoid curl commands and guesswork.

Reply:
With your new example files, this makes a lot of sense.